index.html

```diff
@@ -1861,67 +1861,73 @@ <h3>Lifecycle Details</h3>
         <p>
 Section <a href="#ecosystem-overview"></a> provided an overview of the
 <a>verifiable credential</a> ecosystem. This section provides more detail about
-how the ecosystem is envisaged to operate, as follows:
+how the ecosystem is envisaged to operate:
         </p>
 
-        <ol>
-          <li>
-The <a>verifier</a> defines its policies for accepting
-<a>verifiable credentials</a> from <a>holders</a> for its supported actions.
-          </li>
+        <figure id="life-cycle-details">
+          <img style="margin: auto; display: block; width: 85%;"
+         src="diagrams/ecosystemdetail.svg" alt="diagram showing how
+         credentials flow from issuer to holder, from and optionally 
+         from one holder to another; and how
+         presentations flow from holder to verifier, where
+         three parties can use information from a logical
+         verifiable data registry">
+          <figcaption style="text-align: center;">
+            The roles and information flows for this specification.
+          </figcaption>
+        </figure>
+
+        <ul>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> <a>issues</a> a <a>verifiable credential</a> to a <a>holder</a>.
+Issuance always occurs before any other actions involving this credential. 
```

Comment:
`involving this credential`
-> `involving a credential`

Reply:
done.